src/modules/Core/Wallets/api.js

```diff
@@ -45,6 +56,26 @@ export const getReceiveAddress = (wallet: EdgeCurrencyWallet, currencyCode: stri
   return wallet.getReceiveAddress ? wallet.getReceiveAddress({ currencyCode }) : Promise.resolve(dummyEdgeReceiveAddress)
 }
 
+export const makeSpendInfo = (paymentProtocolInfo: EdgePaymentProtocolInfo): Promise<EdgeSpendInfo> => {
+  const { domain, memo, merchant, spendTargets } = paymentProtocolInfo
```

Comment:
You also have the param `nativeAmount` inside `paymentProtocolInfo` which is already the total amount to spend so you can use that instead of reducing it inside `src/modules/UI/scenes/SendConfirmation/reducer.js`

Reply:
ok